_posts/current/blog/2022-08-18-android-13-now-available-on-qualcomm-reference-boards-rb5-and-rb3.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: Android 13 now available on Qualcomm Reference Boards RB5 and RB3
```

Comment:
Please could you add 'Robotics' before 'Reference' here? Nico was insistent.

Reply:
Ah ok sure no problem